tasks/schema.json

```diff
@@ -11,6 +11,9 @@
     "flag_hash": {
       "type": "string"
     },
+    "difficulty_estimation": {
+      "type": "string"
```

Comment:
Define it as enum, then you can remove your python script changes, because i think our schema verification handles it